INSTALL.md

```diff
@@ -163,6 +163,12 @@ Pandoc can be run through
 [GitHub Actions](https://github.com/features/actions).  For some
 examples, see <https://github.com/pandoc/pandoc-action-example>.
 
+## GitLab CI/CD
+
+Pandoc can be run through
+[GitLab CI/CD](https://about.gitlab.com/stages-devops-lifecycle/continuous-integration/).  For some
+examples, see <https://gitlab.com/pandoc/pandoc-ci-example>.
```

Comment:
Please respect the < 80 column line lengths except where longer lines are unavoidable.

Reply:
@Veratyr Links can make it hard to have sane line lengths, but if you enter them as references with the actual link content of of line with the text it is a lot easier to keep the source readable.